public/v4/apps/posthog.yml

```diff
@@ -0,0 +1,333 @@
+captainVersion: 4
+services:
+    $$cap_appname-db:
+        image: postgres:12-alpine
+        environment:
+            POSTGRES_USER: $$cap_postgres_user
+            POSTGRES_DB: $$cap_postgres_db
+            POSTGRES_PASSWORD: $$cap_postgres_password
+        volumes:
+            - $$cap_appname-postgres-data:/var/lib/postgresql/data
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+
+    $$cap_appname-redis:
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        restart: on-failure
+        image: redis:6.2.7-alpine
+        command: sh -c "redis-server --maxmemory-policy allkeys-lru --maxmemory 200mb"
+
+    $$cap_appname-redis7:
+        image: redis:7.2-alpine
+        restart: on-failure
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        command: sh -c "redis-server --maxmemory-policy allkeys-lru --maxmemory 200mb"
+
+    $$cap_appname-zookeeper:
+        image: zookeeper:3.7.0
+        restart: on-failure
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+        volumes:
+            - $$cap_appname-zookeeper-datalog:/datalog
+            - $$cap_appname-zookeeper-data:/data
+            - $$cap_appname-zookeeper-logs:/logs
+
+    $$cap_appname-clickhouse:
+        restart: on-failure
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+            dockerfileLines:
+                - FROM andremoura/helper:1.0 AS helper
```

Comment:
I just realized that temporary image for some files, why not inline everything here?
https://github.com/AndrMoura/posthog-caprover/blob/main/helper_files.sh

Something like this:
```
FROM alpine AS build 
WORKDIR /
RUN curl -L https://github.com/PostHog/posthog/tarball/c25b571388a9ad05d529a14daee8f49819280496 | tar -xz --wildcards --strip-components=2 -C "./" '*/posthog/idl/*'
RUN curl -L https://github.com/PostHog/posthog/tarball/c25b571388a9ad05d529a14daee8f49819280496 | tar -xz --wildcards --strip-components=2 -C "$DOCKER_DIR" '*/docker/*'

Reply:
I over-complicated things a bit there! Fixed! :)